notebooks/20_real_world_application__graph_space.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<p>define acronym GPCA? (and GGPCA)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>